ddtrace/contrib/aiomysql/patch.py

```diff
@@ -21,7 +21,7 @@
 
 CONN_ATTR_BY_TAG = {
     net.TARGET_HOST: "host",
-    net.TARGET_PORT: "port",
+    "network.destination.port": "port",
```

Comment:
Since we set `network.destination.port` in a few places we should create a constant for it. For consistency we can define it in https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/ext/db.py

Reply:
This task was my "get your legs under you" task, which I noticed a bit late was intentionally underscoped (otherwise this would be a draft 🙁 ).

The idea of literals instead of a separate constant was to avoid having a "temporary" constant that holds the new name until all integrations are switched to the new tag.

Since my legs are now _somewhat_ under me, I have a follow-up task set aside to catch up for the rest of the integrations (including the missing ~SQL modules), which I can handle in the same PR just by switching the value of the existing constant at the end of switching everything to the literal (this way the last commit is just a big `sed` for me, but I can still run tests locally along the way).

Reply:
`out.port` is now totally replaced in all contrib modules using it